docs/source/building-supergraphs/subgraphs-apollo-server.mdx

```diff
@@ -34,16 +38,21 @@ const resolvers = {
   }
 };
 
-const server = new ApolloServer({
-  typeDefs,
-  resolvers,
-});
+async function startApolloServer() {
+  const server = new ApolloServer({
+    typeDefs,
+    resolvers,
+  });
 
-server.listen(4001).then(({ url }) => {
-    console.log(`🚀 Server ready at ${url}`);
-});
+  const { url } = await startStandaloneServer(server);
+  console.log(`🚀  Server ready at ${url}`);
+}
 ```
 
+</MultiCodeBlock>
+
+> Above, we wrap our schema in the `gql` tag from the `graphql-tag` package, converting our schema into an AST (i.e., `DocumentNode`). We do this because the [`buildSubgraphSchema`](../api/apollo-subgraph/#buildsubgraphschema) function requires the schema we pass in to be a `DocumentNode`.
```

Comment:
This is a little confusing since we're not using `buildSubgraphSchema` yet and at this state `gql` isn't required. It might be helpful to also note that this is preemptive and that Apollo Server doesn't actually have this requirement.
```suggestion
> Above, we wrap our schema in the `gql` tag from the `graphql-tag` package, converting our schema into an AST (i.e., `DocumentNode`). While Apollo Server can accept a `string` (or `DocumentNode`) for its `typeDefs`, the [`buildSubgraphSchema`](../api/apollo-subgraph/#buildsubgraphschema) function which we will use below requires the schema we pass in to be a `DocumentNode`.
```